1.data-exploration/scripts/2.test-train_split.py

```diff
@@ -164,7 +164,7 @@ def save_dataframe(df, file_path: pathlib.Path):
 # load in the data
 
 # create dataframe containing the genes that passed an initial QC (see Pan et al. 2022) and a saturated signal qc, then extracting their corresponding gene label
-gene_dict_df = pd.read_csv("../0.data-download/data/CRISPR_gene_dictionary.tsv", delimiter='\t')
+gene_dict_df = pd.read_parquet("../0.data-download/data/CRISPR_gene_dictionary.tsv", delimiter='\t')
```

Comment:
Will this cause an error? It appears that you're trying to load a TSV file instead of a Parquet file.